test/libsolidity/syntaxTests/literal_comparisons.sol

```diff
@@ -0,0 +1,10 @@
+contract test {
+    uint x = 2;
```

Comment:
Sorry, but you need int8 here, this test did not fail before your change. Let me fix it, so that this can be finally merged.